LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_AnalogInput.TcPOU

```diff
@@ -15,9 +15,24 @@ VAR_INPUT
     fTermMax: LREAL;
     // The fReal value correlated with the terminal's min value
     fTermMin: LREAL;
+    // Value to scale the end result to
+    {attribute 'pytmc' := '
+        pv: RES
+        io: io
+    '}
+    fResolution : LREAL := 0.1;
```

Comment:
This should probably default to 1 for backcompat

Reply:
agreed.